docs/concepts/services-networking/networkpolicies.md

```diff
@@ -93,6 +58,10 @@ So, the example NetworkPolicy:
 
 1. allows connections to tcp port 6379 of "role=db" pods in the "default" namespace from any pod in the "default" namespace with the label "role=frontend"
 2. allows connections to tcp port 6379 of "role=db" pods in the "default" namespace from any pod in a namespace with the label "project=myproject"
+{% endcapture %}
 
-See the [NetworkPolicy getting started guide](/docs/getting-started-guides/network-policy/walkthrough) for further examples.
+{% capture whatsnext %}
+*    For configuration instructions, see [Configuring Namespace Isolation](/docs/tasks/administer-cluster/configure-namespace-isolation).
+*    For more NetworkPolicy examples, see the [NetworkPolicy getting started guide](/docs/getting-started-guides/network-policy/walkthrough).
+{% endcapture %}
```

Comment:
Add the include template line to the end of the doc.

{% include templates/concept.md %}